### ISSUE TITLE
Storing Groceries Challange

### ISSUE THREAD

**Issue by MatthijsBurgh** [tue-robotics/tue_robocup#746]

**Comment by PetervDooren** [comment]
TODO

- [ ] Add a placement designator to place objects next to similar objects
- [ ] Make pre-grasp position dependent on the height of the place location
- [ ] Clean up code

optional
- [ ] Open door
- [ ] Pick up heavy object

**Comment by MatthijsBurgh** [comment]
@PetervDooren Why only at high shelves? Just do it always. Should be easier right?

**Comment by PetervDooren** [comment]
@MatthijsBurgh Correct, for this challenge the robot should always turn sideways. Therefore it can be hardcoded in the challenge, which is what I intend to do. 

If we want this functionality in the gpsr it would take more work though
